src/components/BuySellForm/BuySellData/BuySellData.tsx

```diff
@@ -66,7 +68,10 @@ const BuySellData = forwardRef<HTMLDivElement, TBuySellDataProps>(
                                     'lg:hover:absolute': isFloating,
                                 })}
                             >
-                                <Text color='less-prominent' size={labelSize}>{`Rate (1 ${accountCurrency})`}</Text>
+                                <Text color='less-prominent' size={labelSize}>
+                                    {' '}
```

Comment:
What is this extra space for?

Reply:
Apologies, this is a mistake, have fixed this now.